serverless/messaging/api-cli/connect-aws-cli.mdx

```diff
@@ -42,21 +42,21 @@ pip3 install awscli-plugin-endpoint
 
 ## How to configure the AWS-CLI
 
-Now you have installed the AWS-CLI, you need to configure it for use with your Scaleway Messaging and Queuing SQS/SNS namespace.
+Now you have installed the AWS-CLI, you need to configure it for use with Scaleway Messaging and Queuing.
 
 1. Create a file named `~/.aws/config` by running the following command:
     ```
     aws configure set plugins.endpoint awscli_plugin_endpoint
     ```
-2. Open the `~/.aws/config` file you just created in a text editor and edit it as follows (the SNS or SQS URL for your namespace can be found in the console on the list of your [Messaging and Queuing namespaces](https://console.scaleway.com/messaging/namespaces)
+2. Open the `~/.aws/config` file you just created in a text editor and edit it as follows (the SNS or SQS endpoint URL can be found in the console on the **Settings** page for the relevant protocol)
     ```
     [plugins]
```

Comment:
We should probably define 2 profiles here, because credentials are different for SQS and SNS.

Also, because this may conflict with s3 (which uses credentials from our Scaleway IAM), it is probably better to avoid using the default profile.

Then, we could write:

```
    [plugins]
    endpoint = awscli_plugin_endpoint
    [sns]
    sns =
      endpoint_url = https://sns.mnq.fr-par.scaleway.com
    
    [sqs]
    sqs =
      endpoint_url = https://sqs.mnq.fr-par.scaleway.com
```